### PR TITLE
[Feature](agg-state) support write_column_to_pb from DataTypeFixedLengthObjectSerDe

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
## Proposed changes
support write_column_to_pb from DataTypeFixedLengthObjectSerDe
## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

File: be/src/vec/data_types/serde/data_type_string_serde.h [hunk 1]
```diff
@@ -80,6 +80,24 @@ class DataTypeStringSerDe : public DataTypeSerDe {
     template <bool is_binary_format>
     Status _write_column_to_mysql(const IColumn& column, MysqlRowBuffer<is_binary_format>& result,
                                   int row_idx, bool col_const) const;
+
+    virtual bool _is_fixed() const { return false; }
+
+    void write_to_column(IColumn& column, const char* data, size_t size) const {
```
Comment (HappenLee) [inline be/src/vec/data_types/serde/data_type_string_serde.h:86]: should be virtual function，not use `_is_fixed()`。 so wired

**Comment by BiteTheDDDDt** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by Gabriel39 — APPROVED** [review]

**Review by zhangstar333 — APPROVED** [review]

**Pull request merged**